chalice/deploy/packager.py

```diff
@@ -700,6 +714,19 @@ def download_all_dependencies(self, requirements_filename, directory):
                 package_name = match.group(1)
                 raise NoSuchPackageError(str(package_name))
             raise PackageDownloadError(error)
+        stdout = out.decode()
+        match = re.search(self._LINK_IS_DIR_PATTERN, stdout)
+        if match:
+            wheel_package_path = str(match.group(1))
```

Comment:
Is the `str()` needed?  The `stdout` you pass `re.search` is already decoded.

Reply:
That was to make mypy happy.

Reply:
```
chalice/deploy/packager.py:729: error: Argument 1 to "build_wheel" of "PipRunner" has incompatible type "unicode"; expected "str"
```

We do the same thing a few lines above for the same reason: https://github.com/aws/chalice/blob/master/chalice/deploy/packager.py#L701

Reply:
Ah that's right, the whole `AnyStr` thing.  Yeah, shipit then (don't forget to squash your commits).